R/obs-vs-pred-datamapping.R

```diff
@@ -20,7 +20,11 @@ ObsVsPredDataMapping <- R6::R6Class(
       validateIsIncluded(smoother, c("lm", "loess"), nullAllowed = TRUE)
 
       super$initialize(...)
-      self$lines <- lines
+
+      # Apply log10 transformation to lines because
+      # plot is log scaled in by default and geom_abline
+      # requires the log transformed values in input of intercept
+      self$lines <- lapply(lines, log10)

```

Comment:
I think we may want to apply the log10 directly within the `plotXX` functions after checking the `plotConfiguration` scale is included in `Scaling$log` (obs vs pred and res vs pred might be in linear scale).
Otherwise, users can get unwanted log transformation of their diagonal/horizontal lines.

Reply:
If possible could you document the scaling in the `@param lines` and maybe in the pk-ratio vignette